pkg/util/iptables/iptables.go

```diff
@@ -328,10 +328,11 @@ func (runner *runner) checkRuleWithoutCheck(table Table, chain Chain, args ...st
 
 	// Sadly, iptables has inconsistent quoting rules for comments.
 	// Just unquote any arg that is wrapped in quotes.
-	argsCopy := make([]string, len(args))
-	copy(argsCopy, args)
-	for i := range argsCopy {
-		unquote(&argsCopy[i])
+	var argsCopy []string
+	for i := range args {
+		tmpField := args[i]
+		unquote(&tmpField)
+		argsCopy = append(argsCopy, strings.Fields(tmpField)...)
```

Comment:
I've added following debug code right at the beginning of for loop in ```checkRuleWithoutCheck()``` (around line 341)
```go
 glog.V(5).Infof("DBG: fields=%v  args=%v  len(fields)=%d  len(args)+2=%d", fields, args, len(fields), len(args)+2)
```

check out following log.
```
I0819 16:26:39.397647    7288 iptables.go:341] DBG: fields=[-A PREROUTING -m comment --comment "handle ClusterIPs; NOTE: this must be before the NodePort rule
s" -j KUBE-PORTALS-CONTAINER]  args=[-m comment --comment handle ClusterIPs; NOTE: this must be before the NodePort rules -j KUBE-PORTALS-CONTAINER]  len(fiel
ds)=17  len(args)+2=8
```
len(fields) != len(args+2)   
because ```"handle ClusterIPs; NOTE: this must be before the NodePort rules"``` counts as one string in args,
whereas it's 10 strings (each word counting as one) in fields.